master/internal/rm/kubernetesrm/pod.go

```diff
@@ -42,11 +42,16 @@ type podSubmissionInfo struct {
 	taskSpec tasks.TaskSpec
 }
 
-// TODO(mar): messages that are sent by the informer.
+// podStatusUpdate: messages that are sent by the pod informer (TODO: mar).
```

Comment:
nit: i would just put the todo on a new line (also i grep for `TODO(...)` all the time, so that format is helpful to me).